packages/matchbox/src/components/Column/Column.js

```diff
@@ -46,6 +47,7 @@ Column.displayName = 'Column';
 Column.propTypes = {
   children: PropTypes.node,
   width: PropTypes.oneOfType([PropTypes.oneOf(['content']), PropTypes.number]),
+  className: PropTypes.string,
```

Comment:
I think this works. For your use case you could expose `display`

```jsx
import { display } from 'styled-system`;
import { createPropTypes } from '@styled-system/prop-types';

...

const { width, children, className, display } = props;

...

<StyledColumn display={display} />

...

Column.propTypes = {
  ...createPropTypes(display.propNames),
}

```

Reply:
`styled(Box)` already includes `display` so no need to add it to the `StyledColumn` declaration